sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -117,12 +117,16 @@ def __init__(
         **kwargs: Any,
     ) -> None:
         uamqp_transport = kwargs.pop("uamqp_transport", False)
+        amqp_transport: Union[Type[PyamqpTransport], Type[UamqpTransport]] = PyamqpTransport
```

Comment:
May need to do the following since UamqpTransport hasn't been imported:
```suggestion
        amqp_transport: Union[Type[PyamqpTransport], Type["UamqpTransport"]] = PyamqpTransport
```